packages/react-liveness/src/components/FaceLivenessDetector/service/utils/StreamRecorder/StreamRecorder.ts

```diff
@@ -28,6 +32,12 @@ export class StreamRecorder {
     return this.#videoStream;
   }
 
+  setNewVideoStream(stream: MediaStream): void {
```

Comment:
Why can't instantiate a new `StreamRecorder` on camera change?

Reply:
We could do something like that but I figured it would be more complicated. In this solution we can keep track of the initial and the current `MediaRecorder` with the current one feeding data into the initial one. 

With instantiating multiple `StreamRecorders` we would have to instantiate a different type of `StreamRecorder`class which feeds streams into another `StreamRecorder`.